src/components/switch/switch.component.tsx

```diff
@@ -154,6 +154,7 @@ export const Switch = React.forwardRef(
       warning,
       info,
       disabled: disabled || loading,
```

Comment:
```suggestion
      disabled,
```

The whole point of introducing the `loading` prop to `FormField` was to not have to pretend the component is `disabled` when `loading` is set